src/main/java/com/commercetools/sync/utils/StatisticsUtils.java

```diff
@@ -0,0 +1,42 @@
+package com.commercetools.sync.utils;
+
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+
+public class StatisticsUtils {
```

Comment:
Why we are logging stats as json? not others?

Reply:
what do you mean by `others`?

Reply:
We are not formatting other log messages as json, right?

Reply:
because other messages are just string/text messages.

Reply:
or is there something else you mean?